docs/UserGuide.md

```diff
@@ -39,29 +39,31 @@
  
 #### Adding a task or event: `add`
 Adds a task to the to-do list<br>
-Format: `add TASK_NAME [sdate/START_DATE:START_TIME ddate/DEADLINE_DATE:DEADLINE_TIME]`
+Format: `add TASK_NAME [s/START_DATE:START_TIME c/CLOSE_DATE:CLOSE_TIME t/TAG]`
 
-> Date format of START_DATE and DEADLINE_DATE is DD-MM-YYYY:HHMM e.g. 01/10/2016:2100
+> Date format of START_DATE and CLOSE_DATE is DD-MM-YYYY:HHMM e.g. 01/10/2016:2100
 
 * `TASK_NAME` must be unique.
 * If there is no argument, the task will become floating.
 * `START_DATE` refer to the starting date and time of an event. For a task, the timestamp will be automatically saved as start date and time when the task is created. User can input start date and time for events.
-* 
+* `TAG` is for users to write tags for different tasks. Mulitple tags are available by typing `t/TAG t/TAG`.
 
 
 Examples:
-* `add proposal ddate/10-10-2016:2100` <br> Adds a proposal task with a deadline on 10-10-2016 at 21:00
-* `add meeting sdate/01-10-2016:1300 ddate/01-10-2016:2200`<br> Adds a meeting event which start on 1-10-2016 at 1 p.m. and ends at 10 p.m.
+* `add proposal c/10-10-2016:2100` <br> Adds a proposal task with a deadline on 10-10-2016 at 21:00
+* `add meeting s/01-10-2016:1300 c/01-10-2016:2200`<br> Adds a meeting event which start on 1-10-2016 at 1 p.m. and ends at 10 p.m.
 * `add shopping` <br> Adds a floating task named revision test which has not specify the start and end date
+* `add tutorial t/cs2103` <br> Adds a flaoting task named tutorial with a tag CS2013
+* `add quiz t/cs2102 t/easy` <br> Adds a flaoting task named tutorial with a tag CS2012 and easy
 
 #### Deleting a task : `delete`
-Deletes a specific task with the same input task name or index from the to-do list.<br>
+Deletes a specific task by task name or index from the to-do list.<br>
```

Comment:
That's a problem. While we can restrict it such that a number (or a string of it) cannot be a task, we have to also take into account how restricted we want the task name to be. :)